strawberry/flask/tests/test_graphqlview.py

```diff
@@ -0,0 +1,56 @@
+import json
+
+import pytest
+
+from flask import Flask, url_for
+from strawberry.flask.views import GraphQLView
+
+
+def init_app(schema=None):
+    app = Flask(__name__)
+    app.debug = True
+    app.add_url_rule(
+        "/graphql", view_func=GraphQLView.as_view("graphql_view", schema=schema)
+    )
+    return app
+
+
+if __name__ == "__main__":
+    test_app = init_app()
+    test_app.run()
+
+
+@pytest.fixture
+@pytest.mark.usefixtures("schema")
```

Comment:
don't think we need this with functions: https://docs.pytest.org/en/latest/reference.html#pytest-mark-usefixtures

Reply:
@patrick91 please could you explain more on this. I added a fixture for a generic schema that came be used in our test environment. It is located at `tests/fixtures/schema.py`